### PR TITLE
Chore/call purging mechanism

### PR DESCRIPTION
**Pull request by StepanovDaniilOlegovich**
## Description
implemented rise-purge-cached-files call in rise-player-configuration.

## Motivation and Context
Deletes expired cache

## How Has This Been Tested?
In apps Template Editor and Shared Schedules manually with mapping to local builds

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
If any Release Checklist items were intentionally skipped, please provide which ones and the reasons why

### CODE REVIEW

**Comment by stulees** [comment]
@StepanovDaniilOlegovich Curious, have you tested the scenario where you have a schedule of presentation(s) that contains neither Image or Video components? In taking another look at `rise-purge-cache-files.js`, I'm wondering what happens if the _componentCacheNames_ is empty in this line of code - https://github.com/Rise-Vision/common-template/blob/master/src/rise-purge-cached-files.js#L14 .

**Comment by StepanovDaniilOlegovich** [comment]
@stulees thats a good question! thanks for pointing that out!

**Comment by StepanovDaniilOlegovich** [comment]
@stulees added simple catch block with console.log. Is this enough? Should I prepare more advanced error handling strategy? Maybe logging to BQ?

**Comment by stulees** [comment]
@StepanovDaniilOlegovich I think you need to manually test this situation to really see what happens. You will need to force the situation with code to test _componentCacheNames_ being empty. Temporarily change the component cache key values to be arbitrary values so they are not found in cache and _componentCacheNames_ ends up empty in the chain of Promises and see what happens. Thanks

**Comment by StepanovDaniilOlegovich** [comment]
@stulees updated!

File: src/rise-purge-cached-files.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /* eslint-disable no-console, one-var, vars-on-top */
 
 RisePlayerConfiguration.PurgeCacheFiles = (() => {
-  const BASE_COMPONENT_CACHE_KEYS = [ "rise-image", "rise-video" ];
+  const BASE_COMPONENT_CACHE_KEYS = [ "rise-imageaw", "rise-videasdo" ];
```
Comment (stulees) [inline src/rise-purge-cached-files.js:4]: @StepanovDaniilOlegovich looks like you pushed this by mistake

File: src/rise-purge-cached-files.js [hunk 2]
```diff
@@ -18,7 +18,11 @@ RisePlayerConfiguration.PurgeCacheFiles = (() => {
         })
         .then(() => {
           resolve( "done" );
-        });
+        })
+        .catch( err => {
+          console.log( "Error", err );
```
Comment (stulees) [inline src/rise-purge-cached-files.js:23]: @StepanovDaniilOlegovich I think a more descriptive log statement would be helpful, something like `Error trying to purge cache files`

File: src/rise-purge-cached-files.js [hunk 3]
```diff
@@ -18,7 +18,11 @@ RisePlayerConfiguration.PurgeCacheFiles = (() => {
...
+          resolve( "error" );
```
Comment (stulees) [inline src/rise-purge-cached-files.js:24]: @StepanovDaniilOlegovich I think we need unit test coverage for testing different failure states

**Comment by StepanovDaniilOlegovich** [comment]
@stulees ready to review, thanks

**Comment by StepanovDaniilOlegovich** [comment]
@stulees added logging to BQ and test coverage for it

File: src/rise-purge-cached-files.js [hunk 4]
```diff
@@ -2,7 +2,12 @@
 
 RisePlayerConfiguration.PurgeCacheFiles = (() => {
   const BASE_COMPONENT_CACHE_KEYS = [ "rise-image", "rise-video" ];
-  const EXPIRY_TIME = 1000 * 60 * 60 * 24 * 7;
+  const EXPIRY_TIME = 1000 * 60 * 1;
```
Comment (stulees) [inline src/rise-purge-cached-files.js:5]: @StepanovDaniilOlegovich I think you pushed this by mistake, EXPIRY_TIME shouldn't be 1 minute

Comment (StepanovDaniilOlegovich) [reply src/rise-purge-cached-files.js:5]: yes, sorry keep forgetting to delete testing changes, wont happen again
